src/hawkmoth/__init__.py

```diff
@@ -167,7 +167,14 @@ def _get_filenames(self):
                                 location=(self.env.docname, self.lineno))
             return []
 
-        return [os.path.abspath(os.path.join(self.env.config.hawkmoth_root, filename))]
+        filename = os.path.abspath(os.path.join(self.env.config.hawkmoth_root, filename))
+
+        # Warn if the file is not found, but pass it to clang anyway
```

Comment:
Hmm, could we let it fail during the parse instead and, in the `except`, figure out the specific cause and adjust the error message?

Reply:
Can do.

Reply:
Force pushed an update